bloom/infra/spire_api_paging.py

```diff
@@ -0,0 +1,98 @@
+from logging import getLogger
```

Comment:
This is code I got from another project -- I don't really know how to handle "helpers" in Python code architecture. It felt wrong to have it in the main class SpireService, however, I still have a bad feeling about it. Wdyt?